src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -95,6 +96,30 @@ const urgentCareDialogLink = (
   </div>
 );
 
+/**
+ * Helper method to display an urgent care alert dialog
+ *
+ * @param {object} state currentQuery
+ */
+const showDialogUrgCare = currentQuery => {
+  if (
+    (currentQuery.facilityType === LocationType.URGENT_CARE &&
+      currentQuery.serviceType === 'NonVAUrgentCare') ||
+    currentQuery.facilityType === LocationType.URGENT_CARE_FARMACIES
+  ) {
+    return urgentCareDialogLink;
+  }
+
+  if (
+    currentQuery.facilityType === LocationType.CC_PROVIDER &&
+    currentQuery.serviceType === CLINIC_URGENTCARE_SERVICE
+  ) {
+    return urgentCareDialogLink;
+  }
+
+  return null;
+};
```

Comment:
Similar to my other suggestion, this could possibly be combined with `urgentCareDialogLink` into its own component